datafusion/physical-expr/src/expressions/binary/kernels_arrow.rs

```diff
@@ -391,31 +398,159 @@ pub(crate) fn subtract_dyn_decimal(
     decimal_array_with_precision_scale(array, precision, scale)
 }
 
-pub(crate) fn multiply_dyn_decimal(
+/// Remove this once arrow-rs provides `multiply_fixed_point_dyn`.
+fn math_op_dict<K, T, F>(
```

Comment:
Copied from arrow-rs. Once `multiply_fixed_point_dyn` is provided by arrow-rs. We can remove this.

Reply:
I couldn't find any reference to `multiply_fixed_point_dyn` in arrow-rs -- is there an existing ticket?

https://github.com/search?q=repo%3Aapache%2Farrow-rs%20multiply_fixed_point_dyn&type=code